portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,7 +9,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
-@WebServlet("/hello")
+@WebServlet("/artists")
 public class HelloWorldServlet extends HttpServlet {
```

Comment:
Rename `HelloWorldServlet` to something matching what it is now. Also update the comment.